apps/evm/src/components/Tooltip/index.tsx

```diff
@@ -1,22 +1,87 @@
-import { Global } from '@emotion/react';
-import MuiTooltip, { type TooltipProps as MUITooltipProps } from '@mui/material/Tooltip';
+import {
+  Arrow,
+  Content,
+  Portal,
+  Provider,
+  type TooltipContentProps,
+  Tooltip as TooltipPrimitive,
+  type TooltipProps as TooltipPrimitiveProps,
+  Trigger,
+} from '@radix-ui/react-tooltip';
+import { Modal } from 'components/Modal';
+import { useBreakpointUp } from 'hooks/responsive';
+import React, { useState } from 'react';
+import { cn } from 'utilities';
 
-import { useStyles } from './styles';
-
-export interface TooltipProps extends Omit<MUITooltipProps, 'children'> {
-  title: string | React.ReactElement;
-  children: React.ReactNode;
+export interface TooltipProps extends TooltipPrimitiveProps {
+  className?: string;
+  content: string | React.ReactElement;
+  side?: TooltipContentProps['side'];
 }
 
-export const Tooltip = ({ children, placement = 'top', ...rest }: TooltipProps) => {
-  const styles = useStyles();
+const TooltipContent = React.forwardRef<
+  React.ElementRef<typeof Content>,
+  React.ComponentPropsWithoutRef<typeof Content>
+>(({ className, sideOffset = 4, ...props }, ref) => (
+  <Portal>
+    <Content
+      ref={ref}
+      onClick={e => e.stopPropagation()}
+      sideOffset={sideOffset}
+      className={cn(
+        'flex z-50 overflow-hidden rounded-xl bg-lightGrey p-2 max-w-75 text-sm text-primary-foreground animate-in fade-in-0 zoom-in-95',
+        'data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[side=bottom]:slide-in-from-top-2',
+        'data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2',
+        className,
+      )}
+      {...props}
+    />
+  </Portal>
+));
+TooltipContent.displayName = Content.displayName;
+
+export const Tooltip = ({
+  className,
+  content,
+  children,
+  delayDuration = 200,
+  side,
```

Comment:
I don't think either of these properties are needed: both should be constant across the codebase to make the UI look consistent.

Reply:
`side` is replacing the `placement` prop that tooltip used to have ([defaults to top](https://github.com/VenusProtocol/venus-protocol-interface/pull/3818/files/1d9bbaf669d7374481a3205c4d4cfa116c1abb4e?diff=split&w=0#diff-0f120b659412e76d2f4b4cd0c78e5b574db4f90a4b8e08e52749aa900695fe55L11)). `delayDuration` is the time that it takes for the tooltip to pop up, removing it would mean using the default from `TooltipPrimitive` which I feel is too long. What do you think?

Reply:
I suggest using constants for both. So instead of using the default delay from `TooltipPrimitive`, pass 0 (or whatever number you feel works best) to it. Same thing for `side`, I'd pass `top` all the time.